### PR TITLE
Add `HeaderCurrency` component

### PR DESCRIPTION
**Pull request by veado**
- [x] Refactor styles of `HeaderLang` to share common styles

# Desktop view
Screenshot from 2020-06-15 19-06-48

# Mobile view
Screenshot from 2020-06-15 19-07-32

### CODE REVIEW

File: src/services/midgard/types.ts [hunk 1]
```diff
@@ -27,3 +28,10 @@ export type PoolsState = {
 export type PoolsStateRD = RD.RemoteData<Error, PoolsState>
 
 export type NetworkInfoRD = RD.RemoteData<Error, NetworkInfo>
+
+export type PoolCurrency = {
+  symbol: string
+  name: string
+  poolData: PoolData
+}
+export type PoolCurrencies = PoolCurrency
```
Comment (veado) [inline src/services/midgard/types.ts:37]: You can ignore types of `PoolCurrency` + `PoolCurrencies`. These are needed to deal with real data for `HeaderCurrency`, but that will be another PR.

File: src/components/header/HeaderCurrency.stories..tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
```
Comment (nomadnic) [inline src/components/header/HeaderCurrency.stories..tsx:1]: File name should be HeaderCurrency.stories.tsx
